tests/integration-tests/tests/pcluster_api/test_api.py

```diff
@@ -180,6 +292,40 @@ def _test_cluster_workflow(region, api_client, create_cluster, request, pcluster
     _test_delete_cluster(region, cluster_operations_client, cluster_name)
 
 
+def _create_script_sequence(event_name, https_url, s3_url):
+    sequence = []
+
+    for i in range(10):
+        if i % 2 == 0:
+            cache_affinity = i % 2
+            url = f"{https_url}&cache_affinity={cache_affinity}"
```

Comment:
nitpick,non-blocking: you can just hardcode 0 in here

Or, if you want to have just the half of the https url with cache, this does not work

Reply:
good catch, introduced after adding s3 urls, fixed